src/argilla/server/schemas/v1/datasets.py

```diff
@@ -148,13 +154,19 @@ class RatingQuestionSettings(BaseModel):
     )
 
 
+QuestionSettings = Annotated[
```

Comment:
What if we import the `QuestionSettings` from `argilla/server/models/questions.py` instead to avoid defining this twice?

Reply:
Hi @alvarobartt, we cannot do that as the `pydantic.BaseModel` in `argilla/server/models/questions.py` are a bit different from the ones in `argilla/server/schemas/v1/datasets.py`.

Reply:
I would align values with what we're doing for [question and fields names, titles, and descriptions](https://github.com/argilla-io/argilla/pull/3005/files#diff-154a4fe702792a783f557b4e95846e0f9f58dc1c430df9afd0e9d2206f2ef918R39): 200 for name, 500 for titles (can correspond to the text) and 1000 for description. 

We can better adjust those numbers later. My concern about lower values is to be too much restrictive.

Reply:
True @gabrielmbmb didn't see the code above, and thought you were just importing those `pydantic.BaseModel`s from the `questions.py` module 👍🏻

Reply:
True @gabrielmbmb didn't see the code above, and thought you were just importing those pydantic.BaseModels from the questions.py module 👍🏻